bin/cloudflare_python/cloudflare_delete_object.py

```diff
@@ -0,0 +1,14 @@
+import sys
+import boto3
+import cloudflare_client
+
+R2_BUCKET_NAME = sys.argv[4]
```

Comment:
if we won't use the argparse library and keep it simple, could you please add all script arguments as constants like this, in this file and others? A little more self-documenting and readable.

Reply:
Agreed. I changed the code to use `argparse`